x/pocketcore/types/service.go

```diff
@@ -17,13 +17,47 @@ import (
 
 const DEFAULTHTTPMETHOD = "POST"
 
+var (
+	chainHttpClient *http.Client
+)
+
 // "Relay" - A read / write API request from a hosted (non native) external blockchain
 type Relay struct {
 	Payload Payload    `json:"payload"` // the data payload of the request
 	Meta    RelayMeta  `json:"meta"`    // metadata for the relay request
 	Proof   RelayProof `json:"proof"`   // the authentication scheme needed for work
 }
 
+func GetChainsClient() *http.Client {
+	if chainHttpClient == nil {
+		InitHttpClient(1000, 1000, 1000)
```

Comment:
can we remove the hardcoded values?

Reply:
they seem to be defined and used on other usages : 	DefaultRPCMaxIdleConns             = 1000
	DefaultRPCMaxConnsPerHost          = 1000
	DefaultRPCMaxIdleConnsPerHost      = 1000

Reply:
I can set them as parameters to the config.json so that the defaults will be applied, so here we will reference the PocketConfig property.
What do you think @oten91?

Reply:
Yeah, a cycle dependency should be avoided , look for the usage of globalRPCTimeout as an example

Reply:
I update it to use the GlobalPocketConfig load from InitConfig on `x/pocketcore/types/config.go`

Reply:
Thanks!